packaging/linux/run_keybase

```diff
@@ -152,6 +173,8 @@ main() {
   # 2) Users have come to expect that run_keybase will restart everything, and
   #    we tell them to do it after updates.
   kill_all
+  mkdir -p "$mountdir"
+  keybase config set mountdir "$mountdir"
```

Comment:
Will this overwrite a user's changes, if they've set this config by hand? Is it possible to make the current behavior the default if nothing is set, rather than having to explicitly set it? (That would also have the advantage of not requiring `run_keybase` as a post-install step.)

Reply:
Also thanks a million for letting me push so many changes on you.

Reply:
> Will this overwrite a user's changes, if they've set this config by hand?

Yes.  I guess that's not desirable.

> Is it possible to make the current behavior the default if nothing is set, rather than having to explicitly set it?

What do you mean by "current behavior": the current behavior of master, or of this PR?  I can certainly change it to something like:

```sh
if ! keybase config get mountdir &> /dev/null; then
  keybase config set mountdir "$mountdir"
fi
```

Would that be better?  There's a small race between the check and the set of course, but it's probably ok.

> That would also have the advantage of not requiring run_keybase as a post-install step.

I think we want to require folks to move over to the new mountpoint system, so I don't think it's bad to require `run_keybase` personally.  Also note that the "mountdir" config option did nothing on Linux until this PR, before it was Windows-only.

Reply:
Sorry yes, I meant the behavior of this PR, the current desired behavior :) What I'm wondering is whether we could modify the client/kbfs Go code to assume a default value that's the same as what we would be passing to `keybase config set` here. Then `run_keybase` wouldn't really need to configure anything at all, and users who've overridden this already wouldn't run into any trouble either?

Reply:
@oconnor663 and I talked offline, and we agreed on 682307f as a good approach for now.  Merging after CI passes.